WordCounterTest/UnitTest1.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Diagnostics;
+using WordCounterProject;
+
+namespace WordCounterTest
+{
+    [TestClass]
+    public class UnitTest1
```

Comment:
What are you testing here?

Reply:
I should have been more clear.  The name of the test class should be the name of the class your testing with Test appended.  So in this case it would be WordCounterTest

Reply:
Oh I gotcha. In this particular case, I would need to rename the test project or else I'd have a namespace conflict.